sdks/python/apache_beam/runners/common.py

```diff
@@ -463,6 +467,7 @@ def __init__(self,
```

Comment:
<!--new_thread; commit:c89749a0d77350dfbd2686db975aa087bafb8f1b; resolved:0-->
Can you clarify this existing comment?  Especially in contrast to `operation_name`.